db/event_helpers.cc

```diff
@@ -98,7 +103,21 @@ void EventHelpers::LogAndNotifyTableFileCreationFinished(
                             table_properties.num_entries)
               << "num_data_blocks" << table_properties.num_data_blocks
               << "num_entries" << table_properties.num_entries
-              << "filter_policy_name" << table_properties.filter_policy_name;
+              << "deleted_keys" << table_properties.num_deletions
+              << "merge_operands" << table_properties.num_merge_operands
```

Comment:
Lets change these to `num_deletions` and `num_merge_operands` to be in sync with the actual property name.